db/bigtable_eth1.go

```diff
@@ -2102,8 +2102,10 @@ func (bigtable *Bigtable) GetAddressInternalTableData(address []byte, search str
 
 	names := make(map[string]string)
 	for _, t := range transactions {
-		names[string(t.From)] = ""
-		names[string(t.To)] = ""
+		if t != nil {
```

Comment:
I require some information here:

Ranging over `transactions` coming from `GetEth1ItxForAddress` cannot return a `t` that is `nil`, can it? `GetEth1ItxForAddress` includes the following check:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/2f7566c921508e73a435184ee5d86c78258e95f3/db/bigtable_eth1.go#L2083-L2087

The same check should be done in `GetEth1TxForAddress`, `GetEth1ERC20ForAddress` and `GetEth1TxForToken` too: 
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/2f7566c921508e73a435184ee5d86c78258e95f3/db/bigtable_eth1.go#L1727
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/2f7566c921508e73a435184ee5d86c78258e95f3/db/bigtable_eth1.go#L2375
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/2f7566c921508e73a435184ee5d86c78258e95f3/db/bigtable_eth1.go#L3278

Right?

If you agree, some of the `if t != nil {` within `db/bigtable_eth1.go` should be obsolete (the ones related to the functions mentioned).

Please check and share your thoughts.